uc-crux-llvm/src/UCCrux/LLVM/Run/Result.hs

```diff
@@ -77,19 +79,26 @@ ppFunctionSummaryTag =
 
 -- NOTE(lb): The explicit kind signature here is necessary for GHC 8.8/8.6
 -- compatibility.
+--
+-- TODO: It would be great to have more provenance information for the
+-- 'Constraints'. What bug does a given constraint help avoid? On what line?
 data FunctionSummary m (argTypes :: Ctx (FullType m))
   = Unclear (NonEmpty (Located Uncertainty))
   | FoundBugs (NonEmpty (Located TruePositive))
   | SafeWithPreconditions DidHitBounds Unsoundness (Constraints m argTypes)
   | SafeUpToBounds Unsoundness
   | AlwaysSafe Unsoundness
 
-data SomeBugfindingResult
-  = forall m arch argTypes.
+data SomeBugfindingResult m arch
```

Comment:
This could use a description of its contents - in particular, why is there a sequence of results?

Reply:
Comment added.